diff --git a/libs/zstbi/src/zstbi.zig b/libs/zstbi/src/zstbi.zig
index 0136151..0aaa09f 100644
--- a/libs/zstbi/src/zstbi.zig
+++ b/libs/zstbi/src/zstbi.zig
@@ -13,8 +13,6 @@ pub fn init(allocator: std.mem.Allocator) void {
 }
 
 pub fn deinit() void {
-    assert(mem_allocator != null);
-    assert(mem_allocations.?.count() == 0);
     mem_allocations.?.deinit();
     mem_allocations = null;
     mem_allocator = null;
